sormas-backend/src/main/java/de/symeda/sormas/backend/person/PersonFacadeEjb.java

```diff
@@ -1391,7 +1391,7 @@ public List<PersonIndexDto> getIndexList(PersonCriteria criteria, Integer first,
 			cq.orderBy(cb.desc(person.get(Person.CHANGE_DATE)));
 		}
 
-		List<PersonIndexDto> persons = QueryHelper.getResultList(em, cq, first, max);
+		List<PersonIndexDto> persons = QueryHelper.getResultList(em, cq, null, null);
```

Comment:
If we do not use `first` and `max` (because it's applied to the ids query), we should not call `QueryHelper` but use `em.createQuery(cq).getResultList()` directly.

Reply:
Done.